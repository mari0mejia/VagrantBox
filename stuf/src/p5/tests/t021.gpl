on akey {
  if(42 == 42)
    print("Test-A, they are equal.");

  print("Confirm If::execute() ends with:");
  print("    if(next!=nullptr) next->execute();");
}

on akey {
  if(42 == 41)
    print("Test-B, should not see this message.");

  print("Confirm If::execute() ends with:");
  print("    if(next!=nullptr) next->execute();");
}

on akey {
  if(42 == 41)
    print("Test-C, should not see this message.");
  else
    print("Test-C, they are not equal.");

  print("Confirm If::execute() ends with:");
  print("    if(next!=nullptr) next->execute();");
}

on akey {
  if(42 == 42)
    print("Test-D, they are equal.");
  else
    print("Test-D, should not see this message.");

  print("Confirm If::execute() ends with:");
  print("    if(next!=nullptr) next->execute();");
}
