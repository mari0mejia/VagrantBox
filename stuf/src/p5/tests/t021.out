gpl.cpp::main()
  input file(tests/t021.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().
print[3]: Test-A, they are equal.
print[5]: Confirm If::execute() ends with:
print[6]:     if(next!=nullptr) next->execute();
print[13]: Confirm If::execute() ends with:
print[14]:     if(next!=nullptr) next->execute();
print[21]: Test-C, they are not equal.
print[23]: Confirm If::execute() ends with:
print[24]:     if(next!=nullptr) next->execute();
print[29]: Test-D, they are equal.
print[33]: Confirm If::execute() ends with:
print[34]:     if(next!=nullptr) next->execute();

User quit program. Printing the symbol table.
